neighsyncd/neighsync.cpp

```diff
@@ -141,6 +142,20 @@ bool NeighSync::isLinkLocalEnabled(const string &port)
 {
     vector<FieldValueTuple> values;
 
+    if(m_cfgDeviceMetadataTable.get("localhost", values))
+    {
+        auto it = std::find_if(values.begin(), values.end(), [](const FieldValueTuple &t)
+                               { return t.first == "global_ipv6_link_local_neighbors"; });
+        if (it != values.end())
+        {
+            if (it->second == "enable")
+            {
```

Comment:
What if we have global IPv6 LLA status is enabled but port level IPv6 LLA status is disabled? do we handle this case?

Reply:
We do not handle this case - if global IPv6 LLA status is enabled, we consider the IPv6 LLA status as enabled for each port, regardless of the actual setting